tasks/prepare-binaries.js

```diff
@@ -2,6 +2,8 @@ const path = require('path')
 const fs = require('fs')
 const fsExtra = require('fs-extra')
 const readline = require('readline')
+
+process.stdin.unref()
```

Comment:
```suggestion
// When creating a readline.Interface using stdin as input, the program will not terminate until it receives EOF (Ctrl+D on Linux/macOS, Ctrl+Z followed by Return on Windows).
// If you want your application to exit without waiting for user input, you can unref the standard input stream.
// Reference: https://nodejs.org/api/readline.html#readline_readline
process.stdin.unref()
```

Reply:
Can we close it after `readline.createInterface`?

Reply:
Yes of course.